config/settings/local.py

```
@@ -44,6 +44,23 @@
 
 RUNSERVERPLUS_POLLER_RELOADER_TYPE = "watchdog"
 
+# SQL Logging for uvicorn (replaces runserver_plus --print-sql)
+# This provides the same SQL query debugging when using ASGI/uvicorn
+LOGGING = {
+    'version': 1,
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)